lib/bundler/settings.rb

```diff
@@ -138,7 +138,7 @@ def path
       if path = self[:path]
         "#{path}/#{Bundler.ruby_scope}"
       else
-        Bundler.rubygems.gem_dir
+        Pathname.new(File.dirname(__FILE__)).join(".bundle/gems")
```

Comment:
`Bundler.ruby_scope` is a string like `ruby/1.9.1`; why would this be necessary in the path?